### PR TITLE
WD-1708 replaces Canonical OpenStack, Kubernetes and Ceph logos in `/managed`

### PR DESCRIPTION
**Pull request by pedoch**
## Done

- Replaced logos for Canonical OpenStack, Kubernetes and Ceph.
- Reformated the grid items to be cards and increased the row-gap.

## QA

- Navigate to https://ubuntu-com-12618.demos.haus/managed and reference [WD-1708](https://warthogs.atlassian.net/browse/WD-1708) and screenshot section.

## Issue / Card

- [WD-1708](https://warthogs.atlassian.net/browse/WD-1708)

## Screenshots

Previous look:
image

New look:
image

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

[WD-1708]: https://warthogs.atlassian.net/browse/WD-1708?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[WD-1708]: https://warthogs.atlassian.net/browse/WD-1708?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by pedoch** [comment]
@tbille please take a look at this when you can.

File: static/sass/styles.scss [hunk 1]
```diff
@@ -1143,3 +1143,18 @@ hr.p-separator.is-shallow {
...
+  @include vf-b-row-reset;
+
+  @supports (display: grid) {
+    row-gap: 2rem;
+
+    @media (max-width: $threshold-6-12-col) {
+      row-gap: 0;
+    }
+  }
+}
```
Comment (tbille) [inline static/sass/styles.scss:1160]: why did you need to increase this gap?

File: static/sass/styles.scss [hunk 2]
```diff
@@ -1143,3 +1143,18 @@ hr.p-separator.is-shallow {
 .no-border::after {
   content: none !important;
 }
+
+// Custom styles for /managed/index
+// Extend row with increaded row gap
```
Comment (tbille) [inline static/sass/styles.scss:1148]:
```suggestion
// Extend row with increased row gap
```

File: static/sass/styles.scss (hunk 1 above)
Comment (pedoch) [reply static/sass/styles.scss:1160]: The gap was specified in the task. Didn't want to make it an inline style.

**Review by tbille — APPROVED** [review]
Looks good. I just left a small comment for a typo. Once it's fixed, feel free to merge.
